src/services/logAnalyticsSolution/connections.ts

```diff
@@ -47,6 +52,33 @@ export default ({
     }
   }
 
+  /**
+   * Find log analytics workspace related to this log analytics solutions instance
+   */
+  const workspaces: {
+    name: string
+    data: { [property: string]: RawAzureLogAnalyticsWorkspace[] }
+  } = data.find(({ name }) => name === services.logAnalyticsWorkspace)
+
+  if (workspaces?.data?.[region]) {
+    const workspacesInRegion: RawAzureLogAnalyticsWorkspace[] = workspaces.data[
+      region
+    ].filter(({ id: logAnalyticsWorkspaceId }: RawAzureLogAnalyticsWorkspace) =>
+      caseInsensitiveEqual(logAnalyticsWorkspaceId, workspaceResourceId)
+    )
+
+    if (!isEmpty(workspacesInRegion)) {
+      for (const rg of workspacesInRegion) {
+        connections.push({
+          id: rg.id,
```

Comment:
what is `rg.id` here?

Reply:
ahh now I see, it seems that maybe the variable is poorly named but not a big deal.